cmd/bgpmon/cmd/close.go

```diff
@@ -21,7 +22,7 @@ func closeSess(cmd *cobra.Command, args []string) {
 	if bc, clierr := newBgpmonCli(bgpmondHost, bgpmondPort); clierr != nil {
```

Comment:
I still think this should be changed to have the declarations up top. It keeps the code cleaner, more consistent, and we don't really need scoping protection.

Reply:
The [code review document](https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow) explicitly calls for indenting only the error flow.  I've got no strong opinions either way though.